container/blockchain.go

```diff
@@ -109,6 +111,32 @@ func (bc *blockchain) EnsureConsensusWorking(geths []Ethereum, t time.Duration)
 	return err
 }
 
+func (bc *blockchain) RemoveValidators(candidates []Ethereum) error {
+	var newValidators []Ethereum
+
+	for _, v := range bc.validators {
+		istClient := v.NewIstanbulClient()
+		isFound := false
+		for _, c := range candidates {
+			if err := istClient.ProposeValidator(context.Background(), c.Address(), false); err != nil {
+				return err
+			}
+			if v.ContainerID() == c.ContainerID() {
+				isFound = true
+			}
+		}
+		if !isFound {
+			newValidators = append(newValidators, v)
+		}
+	}
+
+	// FIXME: It is not good way to wait validator vote out candidates
+	<-time.After(time.Duration(math.Pow(2, float64(len(candidates)))*7) * time.Second)
```

Comment:
How about letting it be a function parameter? The caller can determine the waiting time.